client/engine/chainservice/mockchain.go

```diff
@@ -0,0 +1,71 @@
+package chainservice
+
+import (
+	"github.com/statechannels/go-nitro/protocols"
+	"github.com/statechannels/go-nitro/types"
+)
+
+// MockChain provides an interface which simulates a blockchain network. It is designed for use as a central service which multiple
+// ChainServices connect to with Go chans.
+// It keeps a record of of holdings and adjudication status for each channel, accepts transactions and emits events.
```

Comment:
```suggestion
//
// It keeps a record of of holdings and adjudication status for each channel, accepts transactions and emits events.
```
required if a paragraph break is intended here.